x-pack/plugins/ml/public/application/trained_models/models_management/models_list.tsx

```diff
@@ -769,6 +786,9 @@ export const ModelsList: FC<Props> = ({
           modelIds={modelIdsToDelete}
         />
       )}
+      {showTestFlyout === null ? null : (
+        <TestTrainedModelFlyout model={showTestFlyout} onClose={() => setShowTestFlyout(null)} />
```

Comment:
```suggestion
        <TestTrainedModelFlyout model={showTestFlyout} onClose={setShowTestFlyout.bind(null, null)} />
```

Reply:
Updated in https://github.com/elastic/kibana/pull/128359/commits/2a9f47840a7e077a838e19e46673fad819b0f80c